packages/flutter_tools/gradle/src/test/kotlin/DependencyVersionCheckerTest.kt

```diff
@@ -254,6 +297,16 @@ object MockProjectFactory {
         // Project path
         every { mockProject.rootDir.path } returns FAKE_PROJECT_ROOT_DIR
 
+        // SDK
+        val actionSlot = slot<Action<Project>>()
+        val mockApplicationExtension = mockk<ApplicationExtension>()
+        every { mockProject.afterEvaluate(capture(actionSlot)) } answers {
+            actionSlot.captured.execute(mockProject)
+            return@answers Unit
+        }
+        every { mockProject.extensions.findByName("android") } returns mockApplicationExtension
```

Comment:
If we don't want to use a constant to find the android extension then consider adding a comment in `DependencyVersionChecker.kt` that points here and says if the string updates then the `MockProjectFactory` also needs to update.